ecommerce/docs/api/data-models.md

```
@@ -926,43 +925,43 @@ CREATE UNIQUE INDEX idx_user_balance_user ON user_balance(user_id);
+| amount | BIGINT | NOT NULL | 변동금액 | 적립(+), 사용(-), 소멸(-) |
 | balance_before | BIGINT | NOT NULL | 거래전잔액 | - |
 | balance_after | BIGINT | NOT NULL | 거래후잔액 | - |
 | order_id | BIGINT | NULL, FK → ORDER(id) | 주문 ID | 주문 관련 거래시만 |
-| description | VARCHAR(255) | NOT NULL | 설명 | "주문 결제", "포인트 충전" 등 |
+| description | VARCHAR(255) | NULL | 설명 | "구매 적립 (5%)", "주문 할인 적용" 등 |
 | is_active | BOOLEAN | NOT NULL, DEFAULT TRUE | 활성화 여부 | - |
 | created_at | TIMESTAMP WITH TIME ZONE | NOT NULL, DEFAULT CURRENT_TIMESTAMP | 생성일시 | - |
 | created_by | BIGINT | NOT NULL | 생성자 | - |
 
 **DDL**:
 ```sql
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)